cli/cmd/plugin/ui/web/tanzu-ui/src/shared/components/ConfigBanner/ConfigBanner.tsx

```diff
@@ -0,0 +1,17 @@
+// React imports
+import React from 'react';
+
+// Library imports
+import { CdsButton } from '@cds/react/button';
+import './ConfigBanner.scss';
+
+function ConfigBanner() {
+    return (
+        <div className="banner-container">
+            <p>Simple configuration</p>
+            <CdsButton action="outline" className='btn'>Use advanced configuration</CdsButton>
```

Comment:
We can hide or remove the advanced configuration button for now

Reply:
hided it